RELEASE.md

```diff
@@ -0,0 +1,94 @@
+  -   [mayadataio/kubera-litmus-subscriber](https://cloud.docker.com/u/mayadataio/repository/docker/mayadataio/kubera-litmus-subscriber)
+
+- KUBERA-PROPEL IMAGES
+  -   [mayadataio/kubera-propel-webapp](https://cloud.docker.com/u/mayadataio/repository/docker/mayadataio/kubera-propel-webapp)
+  -   [mayadataio/kubera-propel-server](https://cloud.docker.com/u/mayadataio/repository/docker/mayadataio/kubera-propel-server)
+  -   [mayadataio/kubera-propel-agent-subscriber](https://cloud.docker.com/u/mayadataio/repository/docker/mayadataio/kubera-propel-agent-subscriber)
+  -   [mayadataio/kubera-propel-agent-controller](https://cloud.docker.com/u/mayadataio/repository/docker/mayadataio/kubera-propel-agent-controller)
+  -   [mayadataio/kubera-propel-iscsi-ubuntu](https://cloud.docker.com/u/mayadataio/repository/docker/mayadataio/kubera-propel-iscsi-ubuntu)
+  -   [mayadataio/kubera-propel-fio](https://cloud.docker.com/u/mayadataio/repository/docker/mayadataio/kubera-propel-fio)
+  -   [mayadataio/kubera-action](https://cloud.docker.com/u/mayadataio/repository/docker/mayadataio/kubera-action)
+
+- KUBERA-AUTH IMAGE
+  -   [mayadataio/kubera-auth](https://cloud.docker.com/u/mayadataio/repository/docker/mayadataio/kubera-auth)
+
+- AGENT-GATEWAY IMAGES
+  -   [mayadataio/gateway-server](https://cloud.docker.com/u/mayadataio/repository/docker/mayadataio/gateway-server)
+  -   [mayadataio/gateway-client](https://cloud.docker.com/u/mayadataio/repository/docker/mayadataio/gateway-client)
+
+- KUBERA-CORE-UI IMAGE
+  -   [mayadataio/kubera-core-ui](https://cloud.docker.com/u/mayadataio/repository/docker/mayadataio/kubera-core-ui)
+
+- KUBERA-CORE-SERVER IMAGE
+  -   [mayadataio/kubera-core-server](https://cloud.docker.com/u/mayadataio/repository/docker/mayadataio/kubera-core-server)
+
+### Kubera Enterprise Modules
+  - [mayadata-io/kubera-litmus](https://github.com/mayadata-io/kubera-litmus)
+  - [mayadata-io/kubera-metal](https://github.com/mayadata-io/kubera-metal)
+  - [mayadata-io/kubera-auth](https://github.com/mayadata-io/kubera-auth)
+  - [mayadata-io/agent-gateway](https://github.com/mayadata-io/agent-gateway)
+  - [mayadata-io/kubera-core-ui](https://github.com/mayadata-io/kubera-core-ui)
+  - [mayadata-io/kubera-core-server](https://github.com/mayadata-io/kubera-core-server)
+
+### Release Process (Needs to follow in the sequential manner)
+  - It starts with making release branch in each of modules
+    Note- kubera-core-server release branch should be created at last
+    
+  - Change `source_branch` from `master` to `release_branch` in `.circleci/config.yaml` in the release branch 
+    ```bash
+    else
+        tag="$CIRCLE_BRANCH-ci"
+        source_branch="master"
+        build_type="ci"
+    fi;
+    ```
+  - Make GitHub tags with the release version for each of modules
+    Note- kubera-core-server github tag should be created at last
+    
```

Comment:
Can you add a point about image names generation from the github/circleci actions? like,
- If build is made from branch, circleci scripts would pick the image name as <branch_name>-ci,
- If build is made from git tag, circleci scripts would pick the image name as <git tag name>

Reply:
Done @vishnuitta